.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -54,19 +51,22 @@ body:
   attributes:
     label: Steps to reproduce
     description: |
-      List of steps or sample code that reproduces the issue. Having reproducible issues is a prerequisite for contributors to be able to solve them.
-      If you include a minimal reproduction project below, you can detail how to use it here.
+      List of steps or sample code that reproduces the issue.
```

This justification is needed IMO, it was added because people misunderstood MRPs and reproduction steps as a requirement to *prove* that the bug exists only
```suggestion
      List of steps or sample code that reproduces the issue. Having reproducible issues is a prerequisite for contributors to be able to solve them.
```

---

I'll reformulate to something like:
> Contributors need concrete ways to easily reproduce the issue.

---

That's great!